frontend/src/scenes/session-recordings/playlist/sessionRecordingsListLogic.ts

```diff
@@ -264,7 +264,10 @@ export const sessionRecordingsListLogic = kea<sessionRecordingsListLogicType>([
         ],
         showFilters: [
             true,
-            { persist: true, storageKey: 'scenes-session-recordings-playlist-sessionRecordingsListLogic-universal' },
+            {
+                persist: true,
+                storageKey: 'scenes-session-recordings-playlist-sessionRecordingsListLogic-showFilters-universal',
```

Comment:
...but why? We already have a good logic for this 😅 If you are sold on this solution then cool but I think its cleaner to have with all the other settings.

Reply:
Since we're going to rework/simplify this area and we're not certain that this is a great metric anyway, just feels like going with the smallest change now is better.